Mathlib/Data/Nat/Factorial/SuperFactorial.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2023 Moritz Firsching. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Firsching
+-/
+import Mathlib.Data.Nat.Factorial.Basic
+
+/-!
+# Super factorial
+
+This file defines the super factorial `1! * 2! * 3! * ...* n!`.
+
+## Main declarations
+
+* `Nat.superFactorial`: The super factorial.
+-/
+
+
+namespace Nat
+
+/-- `Nat.superFactorial n` is the super factorial of `n`. -/
+@[simp]
```

Comment:
This seems like a bad idea to me, it makes the other lemmas redundant

Reply:
I agree, see also  https://github.com/leanprover-community/mathlib4/pull/6840

Reply:
I'm confused; if you agree, why have you resolved the comment but not removed the `simp`?

Reply:
Ah, I had removed it at the lemma below  instead of here. Fixed.